llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -8728,6 +8744,15 @@ class BoUpSLP::ShuffleCostEstimator : public BaseShuffleAnalysis {
       }
       return TTI::TCC_Free;
     };
+    auto GetVF = [&](Value *V) {
```

Comment:
Better make it a function and use here and in other places

Reply:
Add a `getVF` member function. (see https://github.com/llvm/llvm-project/pull/99606/commits/c40998e669c009b32147d66c8d7af18be23fd2e4)